apps/studio/components/grid/components/header/Header.tsx

```diff
@@ -377,23 +380,24 @@ const RowHeader = ({ table, sorts, filters }: RowHeaderProps) => {
 
   return (
     <div className="flex items-center gap-4">
-      <div className="flex items-center gap-3">
-        <Button type="default" className="px-1" icon={<X />} onClick={deselectRows} />
-        <span className="text-xs text-foreground">
-          {allRowsSelected
-            ? `All rows in table selected`
-            : selectedRows.size > 1
-              ? `${selectedRows.size} rows selected`
-              : `${selectedRows.size} row selected`}
-        </span>
-        {!allRowsSelected && totalRows > allRows.length && (
-          <Button type="link" onClick={() => onSelectAllRows()}>
-            Select all rows in table
-          </Button>
-        )}
-      </div>
-      <div className="h-[20px] border-r border-strong" />
       <div className="flex items-center gap-2">
+        <Tooltip_Shadcn_>
```

Comment:
hmm the order of the buttons feel weird it terms of context
- we've got clear selection (which is to do with row selection)
- followed by export and delete
- then select all and the number of rows (which is to do with row selection)

![image](https://github.com/user-attachments/assets/f0258f8f-d125-495c-8a13-2f9058701c7d)

Reply:
any reason why we're changing from the current order of buttons?
![image](https://github.com/user-attachments/assets/dd40af77-0df7-434d-994f-90d2f68bfa32)